src/plugins/data/common/index_patterns/index_patterns/index_pattern.test.ts

```diff
@@ -30,21 +30,6 @@ import { Field } from '../fields';
 
 import { fieldFormatsMock } from '../../field_formats/mocks';
 
```

Comment:
removed this because it was effectively removing code I wanted to test - the `expandShorthand` araguments